src/tests/ftest/deployment/ior_per_rank.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/python
```

Comment:
We shouldn't add a shebang for non-executable files
```suggestion
```

Reply:
Done.